src/comm/PlanckListener.cc

```diff
@@ -55,3 +64,43 @@ void PlanckListener::onHealthTimeout()
     health_timer.stop();
 }
 
+void PlanckListener::setCharger(bool on)
+{
+    charger_on = on;
+    sendChargerOnOffCommand(on);
+    emit chargerChanged();
+}
+
+void PlanckListener::sendChargerOnOffCommand(bool on)
+{
+    buf_t buf(on ? Relay::R1_ON : Relay::R1_OFF);
+    udp.writeDatagram((const char*) &buf.bytes, sizeof(buf.bytes), QHostAddress("192.168.168.7"), 2101);
```

Comment:
Should the IP address be fixed like this?

Reply:
Probably not, we can parameterize.  But I'd like to do that as part of the "LARS optional" tasking we have (e.g. a LARS-specific section in the general settings page)

Reply:
we had to change the logic a bit on Anafi to send multiple relay cmds because sometime the packets were lost:
https://github.com/planckaero/planck_ctrl/blob/master/src/anafi/AnafiLARSControl.cpp#L27-L56

I fear the same might happen here if we send only one

Reply:
Added a 3x re-send (all sent at once)